src/js/components/stock-movement-wizard/StockMovementVerifyRequest.jsx

```diff
@@ -0,0 +1,338 @@
+      currentPage: 1,
+    };
+
+    this.updateWizardValues = this.updateWizardValues.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchBreadcrumbsConfig();
+    this.props.fetchTranslations('', 'stockMovement');
+
+    if (this.props.stockMovementTranslationsFetched) {
+      this.dataFetched = true;
+
+      this.fetchInitialValues();
+    }
+
+    const {
+      actionLabel, defaultActionLabel, actionUrl, listLabel, defaultListLabel, listUrl,
+    } = this.props.breadcrumbsConfig;
+    this.props.updateBreadcrumbs([
+      { label: listLabel, defaultLabel: defaultListLabel, url: listUrl },
+      { label: actionLabel, defaultLabel: defaultActionLabel, url: actionUrl },
+    ]);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.locale && this.props.locale !== nextProps.locale) {
+      this.props.fetchTranslations(nextProps.locale, 'stockMovement');
+    }
+
+    if (nextProps.stockMovementTranslationsFetched && !this.dataFetched) {
+      this.dataFetched = true;
+
+      this.fetchInitialValues();
+    }
+
+    if (nextProps.breadcrumbsConfig &&
+      nextProps.breadcrumbsConfig !== this.props.breadcrumbsConfig) {
+      const {
+        actionLabel, defaultActionLabel, actionUrl, listLabel, defaultListLabel, listUrl,
+      } = nextProps.breadcrumbsConfig;
+
+      this.props.updateBreadcrumbs([
+        { label: listLabel, defaultLabel: defaultListLabel, url: listUrl },
+        { label: actionLabel, defaultLabel: defaultActionLabel, url: actionUrl },
+      ]);
+    }
+  }
+
+  /**
+   * Returns array of form steps.
+   * @public
+   */
+  getStepList() {
+    let stepList = [];
+    if (this.props.hasPackingSupport) {
+      stepList = [this.props.translate('react.stockMovement.create.label', 'Create'),
+        this.props.translate('react.stockMovement.addItems.label', 'Add items'),
+        this.props.translate('react.stockMovement.edit.label', 'Edit'),
+        this.props.translate('react.stockMovement.pick.label', 'Pick'),
+        this.props.translate('react.stockMovement.pack.label', 'Pack'),
+        this.props.translate('react.stockMovement.send.label', 'Send')];
+    } else {
+      stepList = [this.props.translate('react.stockMovement.create.label', 'Create'),
+        this.props.translate('react.stockMovement.addItems.label', 'Add items'),
+        this.props.translate('react.stockMovement.edit.label', 'Edit'),
+        this.props.translate('react.stockMovement.pick.label', 'Pick'),
+        this.props.translate('react.stockMovement.send.label', 'Send')];
+    }
+    return stepList;
+  }
+
+  /**
+   * Returns array of form's components.
+   * @public
+   */
+  getPageList() {
+    let formList = [];
+    if (this.props.hasPackingSupport) {
+      formList = [
+        CreateStockMovement,
+        AddItemsPage,
+        EditPage,
+        PickPage,
+        PackingPage,
+        SendMovementPage,
+      ];
+    } else {
+      formList = [
+        CreateStockMovement,
```

Comment:
As above - we don't need create and add items pages in this workflow.

Reply:
done